js_runner.go

```diff
@@ -180,7 +184,35 @@ func (runner *JSRunner) Call(inputs ...interface{}) (interface{}, error) {
 				return nil, fmt.Errorf("Couldn't convert %#v to JS: %s", input, err)
 			}
 		}
+
+		completed := make(chan bool)
```

Comment:
Not really sure how using the stoppable Timer means the completed channel is no longer needed. My thinking is that the Go routine still needs a way to be notified to stop since stopping the timer does not notify the channel `timer.C` that this stop has occurred.